pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -327,6 +327,15 @@ func (configgen *ConfigGeneratorImpl) createGatewayHTTPFilterChainOpts(
 	node *model.Proxy, server *networking.Server, routeName string) *filterChainOpts {
 
 	serverProto := model.ParseProtocol(server.Port.Protocol)
+
+	httpProtoOpts := &core.Http1ProtocolOptions{
+		AllowAbsoluteUrl: proto.BoolTrue,
```

Comment:
I know the other PR added this as well, but do we or why do we need this? The reason I ask is this will be applied to everyone, not just people that opt in to HTTP 1.0 so I want to make sure we don't break their gateways, especially since most people don't use http 1.0.

Reply:
not for everyone.. only those that enable it via the env var..

Reply:
Oh this piece.. hmm. we use this for http proxy. Why do you need this in gateway?

Reply:
Oh, it seems you may be right, the `allow_absolute_url` may be not needed here